tasks/main.yml

```diff
@@ -9,7 +9,18 @@
   when: ansible_os_family == "Suse"
 
 - include: agent5.yml
-  when: datadog_agent5
+  when: datadog_agent5 and ansible_os_family != "Windows"
 
 - include: agent6.yml
-  when: not datadog_agent5
+  when: not datadog_agent5 and ansible_os_family != "Windows"
+
+- fail:
+    msg: "The Datadog ansible role does not currently support Agent 5"
+  when: datadog_agent5 and ansible_os_family == "Windows"
+
+- include: pkg-windows.yml
```

Comment:
nit-pick: could we move this at the beginning of the file with the other `pkg-*.yml` task ?